fs_attachment/models/ir_binary.py

```diff
@@ -16,16 +16,14 @@
 class IrBinary(models.AbstractModel):
     _inherit = "ir.binary"
 
-    def _record_to_stream(self, record, field_name):
-        # Extend base implementation to support attachment stored into a
-        # filesystem storage
-        fs_attachment = None
+    def _get_fs_attachment_for_field(self, record, field_name):
         if record._name == "ir.attachment" and record.fs_filename:
-            fs_attachment = record
+            return record
+
         record.check_field_access_rights("read", [field_name])
         field_def = record._fields[field_name]
-        if field_def.attachment and not field_def.compute and not field_def.related:
-            field_attachment = (
+        if field_def.attachment and field_def.store:
```

Comment:
I've changed the condition from "is not computed and not related" to "is stored". For instance, all image_64 and image_128 fields, defined by the image.mixin model, are computed fields but still have an actual record in ir.attachment.

This update may also help address the question raised in #411.